README.md

```diff
@@ -1,42 +1,59 @@
-# About
-Current repository contains web interface for ostis platform ([here](https://github.com/ostis-ai) you can find all ostis modules).
+# SC-Web
```

Comment:
Let's add license as in others projects

Reply:
The license of this project is a topic for discussion. I agree that we should add a license, but we don't have one yet, and it looks like a separate PR anyway 🤔